packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.targets

```
@@ -155,4 +155,11 @@
   <!-- Import WPF Build logic only when we don't import NETFX's WinFX targets -->
   <Import Project="Microsoft.WinFX.targets" Condition="'$(ImportFrameworkWinFXTargets)' != 'true'"/>
 
+  <!--
+    Import Windows Forms targets.
+    These come via the Windows Forms transport package, that can be found under
+    https://github.com/dotnet/winforms/tree/main/pkg/Microsoft.Private.Winforms/sdk
+  -->
+  <Import Project="Microsoft.NET.SDk.WindowsDesktop.WindowsForms.targets" Condition="'$(UseWindowsForms)' == 'true'"/>
```

```suggestion
  <Import Project="Microsoft.NET.Sdk.WindowsDesktop.WindowsForms.targets" Condition="'$(UseWindowsForms)' == 'true'"/>
```

Case fix. Also, please add this before the `WinFX` targets.

---

Thank you